### PR TITLE
Fix code to be compatible with newer version of go-bindata (#37)

### PR DESCRIPTION
**Pull request by rokostik**
This PR fixes the code dealing with assets to work with newer versions of go-bindata (I used https://github.com/go-bindata/go-bindata).

Instead of one generated .go file per asset file, it now generates a `bindata.go` file containing all assets. Getting the contents is no longer compile time safe, but is now done with `.MustAsset()` that panics if the asset is not present.

### CODE REVIEW

File: standalone/standalone.go [hunk 1]
```diff
@@ -42,34 +42,33 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 
 	var mux http.ServeMux
 
-	for resourcePath, val := range standalone.GetResources() {
+	for _, assetName := range standalone.AssetNames() {
+		// the index file will be handled separately
+		if assetName == "index-template.html" {
```
Comment (rokostik) [inline standalone/standalone.go]: Not the prettiest way of handling this, but I think it's not that bad.

**Comment by rokostik** [comment]
Looks like the generated `bindata.go` files are not in line with what `gofmt -s` produces, but regular `gofmt` calls without the `-s` makes no changes.

@jhump do you have any preferences what to do in this case? Should `make checkgofmt` not use the `-s` flag when calling `gofmt` or should the generated files simply be run through `gofmt -s`?

File: standalone/standalone.go [hunk 2]
```diff
@@ -116,7 +115,7 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 	})
 }
 
-var indexTemplate = template.Must(template.New("index.html").Parse(string(standalone.GetIndexTemplate())))
+var indexTemplate = template.Must(template.New("index.html").Parse(string(standalone.MustAsset("index-template.html"))))
```
Comment (jhump) [inline standalone/standalone.go]: How about a constant for this name?

File: webform.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/fullstorydev/grpcui/internal/resources/webform"
 )
 
-var webFormTemplate = template.Must(template.New("grpc web form").Parse(string(webform.WebFormTemplate())))
+var webFormTemplate = template.Must(template.New("grpc web form").Parse(string(webform.MustAsset("webform-template.html"))))
```
Comment (jhump) [inline webform.go]: Hmm, that's gross. So the new gobindata doesn't generate functions you can use? It's a shame to lose the type/compile-time safety and have to revert to runtime string checks...

Maybe add provide such functions, hand-written, and add a test that all of the functions return data w/out panic'ing?

File: webform.go [hunk 4]
```diff
@@ -97,13 +97,13 @@ func WebFormContents(invokeURI, metadataURI string, descs []*desc.MethodDescript
 // that first verifies the CSRF token header before delegating to a
 // RPCInvokeHandler.
 func WebFormScript() []byte {
-	return webform.WebFormScript()
+	return webform.MustAsset("webform.js")
```
Comment (jhump) [inline webform.go]: Per other comment, we should revert this change and just hand-write `WebFormScript` and company if necessary. This package should not need to know internal details like these file names.

**Comment by rokostik** [comment]
Made changes as per comments.

Not sure if I understand your comment regarding `go generate` though. AFAIK the `generate` command only looks for binaries on the system and has nothing to do with modules. Please correct me if I'm mistaken, since I also think it would be good to have the `go-bindata` version baked in somewhere.

Also, the problem with `gofmt` and `bindata.go` (a few comments above) is still present and the build will fail.

**Comment by jhump** [comment]
@rokostik, if you have modules enabled and you do things like `go get`, the versions of those tools will be recorded. That allows you to use `go get` in CI and still have repeatable builds, because the tools you use are also pinned.

If you do the following, it should add a line to `go.mod` and two lines to `go.sum` with the version info:

```
cd go/src/github.com/fullstorydev/grpcui
GO111MODULE=on go get github.com/a-urth/go-bindata
```

Please also add a comment to `go.mod` explaining that this version is needed for `go generate`.

Regarding the fact that the output is not properly formatted per the CI check, you can fix that by changing the `go-bindata-all.sh` script (and any other `go:generate` directives that run `go-bindata`) to actually run `gofmt -w -s` after the generator program is run.

**Comment by rokostik** [comment]
Thank you very much for the patience and all the advice.

I should be good to go ready now.
